features/apply.feature

```diff
@@ -147,7 +147,7 @@ Feature: Apply command
   Scenario: Run apply with invalid stack
     When I run `stack_master apply foo bar`
     Then the output should contain "Could not find stack definition bar in region foo"
-    And the exit status should be 1
+    And the exit status should be 0
```

Comment:
Wouldn't this be 1 based on `if stack_definitions.empty? ... success = false` ?

Reply:
it would be 1 if success returns false, because `exit !!false` in bin/stack_master.rb

Reply:
it will never be 1 since we do not run `bin/stack_master` in this test, which includes the `exit`

Reply:
I think we should just remove this test rather than introduce something that tests for undesirable behaviour.

Reply:
then it should be removed from all of them for consistency 🤷‍♂️

Reply:
While the other tests are not accurate they do reflect the behaviour we want to see.

This test is both inaccurate and does not reflect true behaviour